proxy-node-shared/src/main/java/uk/gov/ida/notification/shared/proxy/VerifyServiceProviderProxy.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ida.notification.shared.proxy;
+
+import uk.gov.ida.jerseyclient.JsonClient;
+import uk.gov.ida.notification.contracts.HubResponseTranslatorDto;
+import uk.gov.ida.notification.contracts.TranslatedHubResponse;
+
+import javax.ws.rs.core.UriBuilder;
+import java.net.URI;
+
+import static uk.gov.ida.notification.shared.Urls.*;
+
+public class VerifyServiceProviderProxy {
+    private final JsonClient jsonClient;
```

Comment:
If it is just the JsonClient we need from ida-utils, is it worth just bringing this class across rather than using the whole lib as a dependency?

Reply:
As discussed, we can that library